src/pages/index.astro

```diff
@@ -91,6 +91,15 @@ const entries: CardProps[] = [
     topic: AcademicTopics.Solid,
     imagePath: "/images/mukpakpak.jpeg",
     githubUrl: "https://github.com/tinarskii/mukpakpak",
+  },
+  {
+    display: "dark",
+    title: "KNACKSAT2 TGPS Sattlelite Open-source hardware",
```

Comment:
its name is a little bit long, it would be trim on screen to `Open-...` but if you OK with it. feel free to mark it resolved.
<img width="1512" alt="Screen Shot 2565-10-22 at 21 43 47" src="https://user-images.githubusercontent.com/37364131/197345360-e8ac18a9-e723-4af9-9bee-17fd626c2208.png">

Reply:
Roger problem, fixing the length of the name now.

Reply:
fixed.